app/controllers/parties_controller.rb

```diff
@@ -1,27 +1,28 @@
-class PartiesController < ActionController::Base
-  before_action :create_movie, only: [:create]
+class PartiesController < ApplicationController
```

Comment:
What's the difference here when removing ::Base?

Reply:
We normally want our controllers to Inherit from ApplicationController not ActionController, only ApplicationController inherits from ActionController::Base